packages/insomnia-app/app/network/network.js

```diff
@@ -61,14 +62,15 @@ import {
 import fs from 'fs';
 import * as db from '../common/database';
 import CACerts from './ca-certs';
+import winCa from 'win-ca';
```

Comment:
We should import `win-ca/api` here instead to avoid the import side-effects (overwriting the node trust store) that `win-ca` does by default:
```suggestion
import winCa from 'win-ca/api';
```

I think we might also need to add `win-ca/api` to `externals` in the insomnia-app and possibly insomnia-inso webpack configuration for this to work.